Assets/Scripts/PlayerMovement/CharacterController2D.cs

```diff
@@ -3,42 +3,46 @@
 
 public class CharacterController2D : MonoBehaviour
 {
-    [SerializeField] private float m_JumpForce = 400f;                            // Force added when the player jumps.
-    [Range(0, .3f)] [SerializeField] private float m_MovementSmoothing = .05f;   // Smoothing for movement.
-    [SerializeField] private bool m_AirControl = false;                          // Can the player steer while jumping.
-    [SerializeField] private LayerMask m_WhatIsGround;                           // What is considered ground.
-    [SerializeField] private Transform m_GroundCheck;                            // Position to check if the player is grounded.
+    [SerializeField] private float _JumpForce = 400f;                            // Force added when the player jumps.
```

Comment:
Dari standar penamaan baris ini "Private instance fields start with an underscore (_) and the remaining text is camelCased.
" huruf setelah underscore tidak kapital